tests/unit-tests/test_memory.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import sys
+from unittest.mock import MagicMock
+
+# Replace `pwndbg.commands` module with a mock to prevent import errors, as well
+# as the `load_commands` function
+module_name = "pwndbg.commands"
+module = MagicMock(__name__=module_name, load_commands=lambda: None)
+sys.modules[module_name] = module
```

Comment:
We do run tests in single sessions currently so this works fine but I believe this would not work if multiple test files would be executed in a single session, since this mock is never reverted.

Reply:
I think this is actually a good thing. In fact, we should probably do something similar to `tests/unit-tests/mocks/gdblib.py` and import it with `mocks.commands` like below (but this doesn't need to be done in this PR).

That being said, I'm wondering why we only need to mock this out now? Why are the other tests working? What error shows up in this test if we don't have this mock?

Reply:
I copied this chunk from the other tests since both the other unit test files have it, but commenting it out doesn't actually seem to break anything, so maybe that can be removed/replaced with an actual mock?

Reply:
Ah got it. You can leave it as is for now, but in the future we should really consolidate or remove this so it's less boilerplate when writing a test.